lib/oli/accounts/schemas/author.ex

```diff
@@ -91,6 +91,23 @@ defmodule Oli.Accounts.Author do
     |> maybe_name_from_given_and_family()
   end
 
+  @doc """
+  Creates a changeset that is used to update an author's profile
+  """
+
+  def update_author_changeset(author, attrs \\ %{}) do
```

Comment:
What do you think about changing the name of this function to `update_changeset` because we are in the Author context.

Same for the User side.